### DISCUSSION TITLE
react 18 的支持

### DISCUSSION THREAD

**Discussion by hlskysong** [jd-opensource/micro-app#560] [Q&A]
基座 vue2 ,
子项目 umi3.5 react17 挂载渲染正常
umi4 react18 挂载正常 显示白屏

react 18 的支持有安排了吗?

**Comment by bailicangdu** [comment]
micro-app版本是多少

**Reply by hlskysong** [reply]
"@micro-zoe/micro-app": "^0.8.10",

**Reply by bailicangdu** [reply]
升级到 [1.0.0-alpha.7](https://www.npmjs.com/package/@micro-zoe/micro-app/v/1.0.0-alpha.7) 试一下

**Reply by wdm-coding** [reply]
react +antd +vite 作为子应用，antd 的下拉菜单 dorpdown 定位异常，无法解决，
